core/templates/services/speech-synthesis-chunker.service.ts

```diff
@@ -161,17 +159,15 @@ export class SpeechSynthesisChunkerService {
       .trim();
   }
 
-  // TODO(#7165): Replace 'any' with the exact type. This has been kept as
-  // 'any' because the type of 'html' needs to be determined correctly.
-  _convertToSpeakableText(html: any): string {
+  _convertToSpeakableText(html: string): string {
     Object.keys(ServicesConstants.RTE_COMPONENT_SPECS).forEach(
       (componentSpec) => {
         this.RTE_COMPONENT_NAMES[componentSpec] =
         ServicesConstants.RTE_COMPONENT_SPECS[componentSpec].frontend_id;
       });
     interface IMathExpressionContent {
-      'raw_latex': String,
-      'svg_filename': String
+      'raw_latex': string,
```

Comment:
@donosco98, Maybe you'll find this helpful for future work!  (I had also missed this :p )